docs/workflow/blocks/db.rst

```diff
@@ -3,3 +3,66 @@ Local Database
 
 The local database block implements an indexed database in the browser storage. 
 The block implements a version of Dexie.js. 
+
+
+Default config
+--------------
+
+.. code-block:: json
+
+    {
+      type: 'db',
+      operation: 'add',
+      adapterName: 'UNKNOWN',
+      schema: 'none',
+      skipFirst: true,
+    }
+
+Supported properties
+--------------------
+- **operation** - the database operation to undertake (See below)
+- **adapterName**  - the name of the adapter
```

Comment:
What is an adapter?

Reply:
Its the standard adapter name we use for all blocks. Schema's were originally designed to be tied to downloadable adapters.